src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/ModuleSpeed.kt

```diff
@@ -43,10 +49,36 @@ object ModuleSpeed : Module("Speed", Category.MOVEMENT) {
                 wait(1)
                 player.downwards(-1f)
             }
-
         }
+    }
+
+    private object HypixelHop : Choice("HypixelHop", modes) {
+        val playerNetworkMovementTick = handler<PlayerNetworkMovementTickEvent> { event ->
 
+            if (event.state != EventState.PRE)
+                return@handler
+
+            if (player.moving) {
+                if (player.isOnGround) {
+                    player.updatePosition(player.x, player.y + 9.1314E-4, player.z)
```

Comment:
Are you sure, that this is going to bypass?

Reply:
if you are talking about player.updateposition, im not quite sure. seeing from his version of hypixelhop, i don't think it's going to cause any problems? on the other hand, the person who made the issue says to use e.getY()...